vocabularies/Common.xml

```diff
@@ -1319,6 +1320,19 @@ Upon modification field control logic is invoked so that meta-information like h
         <Annotation Term="Core.LongDescription" String="TODO: add link to UX documentation on https://experience.sap.com/fiori-design/" />
       </Term>
 
+      <Term Name="mediaUploadURL" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.RequiresType" String="Edm.Stream" />
+        <Annotation Term="Core.IsURL" />
+        <Annotation Term="Core.Description" String="URL for uploading new media content to a Document Management Service" />
+        <Annotation Term="Core.LongDescription">
+          <String>In contrast to the `@odata.mediaEditLink` this URL allows to upload new media content without directly changing a stream property.
+The upload request typically uses POST with `Content-Type: multipart/form-data`.

```

Comment:
But that does not specify how to construct the payload. For example, how is the multipart named that contains the uploaded file? How can a consumer find out?

Reply:
@bernhardthimmel please answer

Reply:
I'll get some input from the colleagues regarding the structure of the payload. We also must see if we need to keep some flexibility here so that we do not need to swamp the place with many different annotations and perhaps have services document this explicitly. I'll get back to this thread once I have some more answers.
@HeikoTheissen I am not quite sure what the 'uploaded file in the payload' would mean. Should we have a short sync tomorrow or on Friday so I can talk you through our scenario?

Reply:
Document the functionality of the POST request, e.g., by writing down an HTML form in the annotation definition or an external document.
```html
<form enctype="multipart/form-data" action="..." method="POST">
 <input type="file" name="GIVE_THE_NAME_HERE"/>
 <input type="submit" name="GIVE_THE_NAME_HERE"/>
</form>
```

Reply:
```suggestion
The upload request typically uses HTTP POST with `Content-Type: multipart/form-data` following RFC 7578.
```